### PR TITLE
TLC migration to the new schedule

### PR DESCRIPTION
**Pull request by iamsahu**
## Notice
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/liquid-collective/liquid-collective-protocol/pulls) for the same update/change?
- [x] Have you assigned this PR to yourself?
- [x] Have you added at least 1 reviewer?
- [ ] Have you updated the official documentation?
- [ ] Have you added sufficient documentation in your code?
- [x] Have you added relevant tests to the official test suite?

## Pull Request Type

- [ ] 💫 New Feature (Breaking Change)
- [ ] 💫 New Feature (Non-breaking Change)
- [ ] 🛠️ Bug fix (Non-breaking Change: Fixes an issue)
- [x] 🕹️ Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)

## Testing

- [x] Have you tested this code with the official test suite?
- [x] Have you tested this code manually?

### CODE REVIEW

**Review by juliaaschmidt — APPROVED** [review]
🚀

**Comment by mischat** [comment]
@iamsahu the tests are passing ...

File: contracts/test/migration/TLC_globalUnlockScheduleMigration.t.sol [hunk 1]
```diff
@@ -47,69 +153,29 @@ contract TlcMigrationTest is Test {
...
+            assertEq(schedule.end, schedulesBefore[i].end);
+            assertEq(schedule.cliffDuration, schedulesBefore[i].cliffDuration);
+            assertEq(schedule.duration, schedulesBefore[i].duration);
+            assertEq(schedule.periodDuration, schedulesBefore[i].periodDuration);
             assertEq(schedule.amount, schedulesBefore[i].amount);
             assertEq(schedule.creator, schedulesBefore[i].creator);
             assertEq(schedule.beneficiary, schedulesBefore[i].beneficiary);
             assertEq(schedule.revocable, schedulesBefore[i].revocable);
             assertEq(schedule.releasedAmount, schedulesBefore[i].releasedAmount);
         }
```
Comment (koderholic) [inline contracts/test/migration/TLC_globalUnlockScheduleMigration.t.sol:173]: Hey @iamsahu I noticed you didn't check to confirm ignoreGlobalUnlock shouldn't change, is this not necessary?

File: scripts/helpers.ts [hunk 2]
```diff
@@ -26,3 +28,48 @@ const tenderlyVerify = async (name: string, contractAddress: string) => {
...
+    data: upgradeData,
+    nonce: txCount,
+  });
+  await tx.wait();
+  console.log("tx >> ", tx);
+  console.log(`${upgrading} proxy upgraded to ${newImplementationAddress} with initialization`);
+}
+
+export async function upgradeTo(deployments, ethers, newImplementation, signer, sendTo, upgrading) {
+  const proxyTransparentArtifact = await deployments.getArtifact("ITransparentUpgradeableProxy");
```
Comment (koderholic) [inline scripts/helpers.ts:63]: Nice one factoring this out. Thank you. 👍👍

**Review by juliaaschmidt — DISMISSED** [review]

**Review by koderholic — APPROVED** [review]
LGTM

**Review by mischat — APPROVED** [review]
LGTM

**Review by juliaaschmidt — APPROVED** [review]

**Review by mischat — APPROVED** [review]
LGTM

**Pull request merged**